Identity/Identity.DataAccess/Models/DbInitializer.cs

```diff
@@ -0,0 +1,54 @@
+using System.Security.Cryptography;
+using System.Text;
+using Identity.DataAccess.Models.Entities;
+
+namespace Identity.DataAccess.Models;
+
+public static class DbInitializer
+{
+    private static UserEntity[] SeedUsersData => CreateSeedUsersData();
+
+    public static async Task SeedDatabase(IdentityDatabaseContext context)
+    {
+        await context.Users.AddRangeAsync(SeedUsersData);
+        await context.SaveChangesAsync();
+    }
+
+    private static UserEntity[] CreateSeedUsersData()
+    {
+        return
+        [
+            new UserEntity
+            {
+                Username = "jonas123",
+                PasswordHash = HashPassword("jonas123"),
+                Email = "jonas123@gmail.com",
+                Roles = ["member"],
```

Comment:
Could use `UserRoles.Member` etc.

Reply:
Identity.Public is not referenced by Identity.DataAccess

Reply:
Hm, wondering if it would be bad to reference

Reply:
Probably not, but the data access layer doesn't necessarily need to know about how the application presents the data to the consumer. I think this will suffice.

Reply:
yeah it's fine